.yarnrc.yml

```diff
@@ -1,7 +1,9 @@
 nodeLinker: node-modules
 
+npmRegistryServer: "https://registry.yarnpkg.com"
+
 plugins:
   - path: .yarn/plugins/@yarnpkg/plugin-outdated.cjs
     spec: "https://mskelton.dev/yarn-outdated/v3"
 
-yarnPath: .yarn/releases/yarn-3.2.1.cjs
+yarnPath: .yarn/releases/yarn-3.8.1.cjs
```

Comment:
Yarn 4 is released if you want to update

https://yarnpkg.com/blog/release/4.0

Reply:
Updated to yarn 4. Looks like it's expecting node 18. I updated references from 16, but I'm not seeing where the github action for test (16.x) is coming from in the build.yml file.

Reply:
https://github.com/Widen/expect-axe-playwright/blob/main/.github/workflows/build.yml#L12

It's in the matrix setup. You can remove 16.x and add 20.x in there.

Reply:
Interesting. I had removed 16. I'll try adding 20.x.
https://github.com/Widen/expect-axe-playwright/pull/29/commits/1fcb716c01bd6972e909acbe034f24ebd2b2c7db

Reply:
Ahh it's a branch protection rule

Reply:
Got it. All is good now. Thanks, @mskelton